applications/advanced-concepts/tags-and-placeholders.md

```diff
@@ -0,0 +1,17 @@
+---
+description: Tags and placeholders in machine translation
+---
+
+# Tags and placeholders
+
+**Tags and placeholders** are pieces of non-translatable code. While tags format and structure texts, placeholders are variables for dynamic content.
+
+## Challenges
+
+- The machine translation can misplace, corrupt or ignore tags and placeholders.
+
+In a phrase like "Susan's car", the genitive apostrophe can appear as a tag. When translated into Spanish, the appearance of the tag in the output introduces an error.
+
+- The machine translation output quality can be degraded when the final content replaces tags and placeholders.
```

Comment:
> - The presence of tags and placehold can degrade the translation quality.
> - The reinsertion of tags and placeholders can degrade the translation quality.

Because there are 2 approaches to tags and placeholders:
1. Let the MT system figure out how to preserve them.
2. Remove them from training data and at run time, and then reinsert them after translation (which is not trivial).

(Maybe this should go above, but phrases better.)